src/config/i18n/en-GB/commands.json

```
@@ -802,6 +802,9 @@
       "notStaff": "You must be a member of our support staff to use this command.",
       "invalidId": "{{- id}} is not a valid guild ID.",
       "logs": "Here are the available logs for your guild:"
+    },
+    "ids": {
+      "message": "Here are the relevant IDs. These are safe to share with our support team.\n\nUser ID: {{userId}} \n Channel ID: {{channelId}} \n Guild ID: {{guildId}}"
```

```suggestion
      "message": "Here are the relevant IDs. These are safe to share with our support team.\n\nUser ID: {{userId}}\nChannel ID: {{channelId}}\nGuild ID: {{guildId}}"
```

Spacing's off a bit.